tests/regression/models/solid_mechanics/regression_isotropic_and_kinematic_hardening.cxx

```diff
@@ -26,76 +26,18 @@
-                                      /*rtol =*/1e-8,
-                                      /*miters =*/100,
-                                      /*verbose=*/false};
-
-  Scalar E = 1e5;
-  Scalar nu = 0.3;
-  SymSymR4 C = SymSymR4::init(SymSymR4::FillMethod::isotropic_E_nu, {E, nu});
-  Scalar s0 = 5;
-  Scalar K = 500;
-  Scalar H = 1000;
-  Scalar eta = 100;
-  Scalar n = 2;
-  auto Erate = std::make_shared<ForceRate<SymR2>>("total_strain");
-  auto Eerate = std::make_shared<ElasticStrainRate>("elastic_strain_rate");
-  auto elasticity = std::make_shared<CauchyStressRateFromElasticStrainRate>("elasticity", C);
-  auto mandel_stress = std::make_shared<IsotropicMandelStress>("mandel_stress");
-  auto isoharden = std::make_shared<LinearIsotropicHardening>("isotropic_hardening", K);
-  auto kinharden = std::make_shared<LinearKinematicHardening>("kinematic_hardening", H);
-  auto sm = std::make_shared<J2StressMeasure>("stress_measure");
-  auto yield =
-      std::make_shared<IsotropicAndKinematicHardeningYieldFunction>("yield_function", sm, s0);
-  auto direction =
-      std::make_shared<AssociativePlasticFlowDirection>("plastic_flow_direction", yield);
-  auto eeprate = std::make_shared<AssociativeIsotropicPlasticHardening>("eeprate", yield);
-  auto eprate = std::make_shared<AssociativeKinematicPlasticHardening>("ep_rate", yield);
-  auto hrate = std::make_shared<PerzynaPlasticFlowRate>("hardening_rate", eta, n);
-  auto Eprate = std::make_shared<PlasticStrainRate>("plastic_strain_rate");
-
-  auto rate = std::make_shared<ComposedModel>("rate",
-                                              std::vector<std::shared_ptr<Model>>{Erate,
-                                                                                  Eerate,
-                                                                                  elasticity,
-                                                                                  mandel_stress,
-                                                                                  isoharden,
-                                                                                  kinharden,
-                                                                                  yield,
-                                                                                  direction,
-                                                                                  eeprate,
-                                                                                  eprate,
-                                                                                  hrate,
-                                                                                  Eprate});
-
-  auto implicit_rate = std::make_shared<ImplicitTimeIntegration>("implicit_time_integration", rate);
-  auto solver = std::make_shared<NewtonNonlinearSolver>(params);
-  auto model = std::make_shared<ImplicitUpdate>("viscoplasticity", implicit_rate, solver);
+  HITParser parser;
```

Comment:
Same comment on helper here and everywhere below.

Reply:
Fixed.